src/Files.App/Filesystem/Drives.cs

```diff
@@ -91,7 +91,8 @@ public static async Task<StorageFolderWithPath> GetRootFromPathAsync(string devi
 					return new StorageFolderWithPath(matchingDrive, rootPath);
 				}
 			}
-			else if (devicePath.StartsWith(@"\\", StringComparison.Ordinal)) // Network share
+			else if (devicePath.StartsWith(@"\\", StringComparison.Ordinal) &&
+				!devicePath.StartsWith(@"\\SHELL\", StringComparison.Ordinal)) // Network share
```

Comment:
```suggestion
			// Network share or Shell folder
			else if (devicePath.StartsWith(@"\\", StringComparison.Ordinal) &&
				!devicePath.StartsWith(@"\\SHELL\", StringComparison.Ordinal)) // Network share
```

Reply:
No. Network share EXCLUDING shell folders.

Reply:
In any way, corrected comment position.